about.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>About Excellence University></title>
+        <link rel="stylesheet" href="style.css">     
+    </head>
+    <body> 
+        <section class="aboutus-header">
+            <nav class="navbar">
+                <a href="index.html">
+                    <img class="logo" src="images/logo.png">
+                </a>
+                <div class="nav-links" id="navLinks">
+                    <i class="fa fa-times" onclick="hideMenu()"></i>
+                    <ul lass="nav-links__list">
+                        <li class="nav-links__item"><a href="index.html" class="nav-links__link">Home</a></li>
+                        <li class="nav-links__item"><a href="about.html" class="nav-links__link">About</a></li>
+                        <li class="nav-links__item"><a href="index.html" class="nav-links__link">Courses</a></li>
+                        <li class="nav-links__item"><a href="index.html" class="nav-links__link">Admission</a></li>
+                    </ul>
+                </div>
+                <i class="fa fa-bars" onclick="showMenu()"></i>
+            </nav>                  
+            <h1 class="heading-primary">About Us</h1>
+        </section>  
+        <div class="aboutus-page">
+            <section class="content-container">
+                <div class="aboutus-subheader">
+                    <h2 class="aboutus-subheading">Transformation through education</h2>
+                </div>
+                <div class="title">
+                    <h2 class="title__copy">Who we are</h2>
+                </div>
+                <div class="about-row">
+                    <div class="about-col">
+                        <img class="about-image" src="images/about.jpg">
+                    </div>
+                    <div class="about-col">
+                        <h2 class="para-title">World's largest university</h2>
+                        <p class=para-copy>Every individual has the potential to create change, whether in their life, their community, or the world. The transformative power of education is what unlocks that potential. Yet, access to high-quality education has been a privilege of the few. From the tried and true to the leading edge. From “for some” to “for all.” By opening the classroom through online learning, edX empowers millions of learners to unlock their potential and become changemakers.</p>
+                        <a href="" class="hero-btn hero-btn--color-red">Explore now</a>
```

Comment:
href is empty, it's invalid html

also, not critical, but it can be just `hero-btn--red`